### PR TITLE
cloud: remove playground

### PR DESCRIPTION
**Pull request by ran-huang**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v7.2 (TiDB 7.2 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v7.0 (TiDB 7.0 versions)
- [ ] v6.6 (TiDB 6.6 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-cloud/tidb-cloud-roadmap.md [hunk 1]
```diff
@@ -62,7 +62,7 @@ The TiDB Cloud roadmap brings you what's coming in the near future, so you can s
   </tr>
   <tr>
     <td>Optimize Playground.</td>
-    <td>Improve interactivity combined with Chat2Query, and guide users to finish the tutorial.</td>
+    <td>{To be updated}</td>
```
Comment (ran-huang) [inline tidb-cloud/tidb-cloud-roadmap.md:65]: @shizn Can you provide some suggestions on what to write in this line?

**Review by Oreoxmt — APPROVED** [review]
LGTM

File: tidb-cloud/tidb-cloud-quickstart.md [hunk 2]
```diff
@@ -76,15 +76,9 @@ In Chat2Query, you can either simply type `--` followed by your instructions to
 
 After running the queries, you can see the query logs and results immediately at the bottom of the page.
 
-## Step 3. Try Playground
+## Step 3. Load sample data
```
Comment (shizn) [inline tidb-cloud/tidb-cloud-quickstart.md:79]: I would suggest let new users try the interactive tutorials in Step3. 
## Step 3. Try Interactive Tutorials. 
Click the ? button on the right bottom of your screen. And click the "Interactive Tutorials". Select one of the pre-built sample datasets and explore TiDB Cloud's features and capabilities.

File: tidb-cloud/tidb-cloud-roadmap.md (hunk 1 above)
Comment (shizn) [reply tidb-cloud/tidb-cloud-roadmap.md:65]: Deliver context-driven tutorials to facilitate a deeper understanding of TiDB and TiDB Cloud for our users.

Comment (ran-huang) [inline tidb-cloud/tidb-cloud-roadmap.md:65]:
```suggestion
    <td>Deliver context-driven tutorials to facilitate a deeper understanding of TiDB and TiDB Cloud for our users.</td>
```

**Review by shizn — APPROVED** [review]

File: tidb-cloud/tidb-cloud-quickstart.md [hunk 3]
```diff
@@ -76,33 +76,14 @@ In Chat2Query, you can either simply type `--` followed by your instructions to
...
-
-4. Add the table filter rules if needed. For the sample data, you can skip this step. Click **Next**.
-
-5. On the **Preview** page, confirm the data to be imported and then click **Start Import**.
-
-The data import process will take several minutes. When the data import progress shows **Finished**, you have successfully imported the sample data and the database schema to your database in TiDB Cloud.
+1. Click the **?** element on the bottom right of the console, and click **Interactive Tutorials**.
+2. In the tutorials list, select a tutorial card to start. For example, **Steam Game Stats**.
+3. Select a TiDB Serverless cluster that you want to use for the tutorial, and click **Import Dataset**. The import process might take approximately 1 minute.
+4. After the sample data is imported, follow the on-screen instructions to complete the tutorial.
```
Comment (Oreoxmt) [inline tidb-cloud/tidb-cloud-quickstart.md:86]:
```suggestion
1. Click **?** in the lower-right corner of the console and click **Interactive Tutorials**.
2. In the tutorials list, select a tutorial card to start. For example, **Steam Game Stats**.
3. Select a TiDB Serverless cluster that you want to use for the tutorial, and click **Import Dataset**. The import process might take approximately one minute.
4. After the sample data is imported, follow the on-screen instructions to complete the tutorial.
```

**Review by Oreoxmt — COMMENTED** [review]
Rest LGTM

**Review by Oreoxmt — APPROVED** [review]

**Pull request merged**